src/core/Akka.Remote.Tests/RemotingSpec.cs

```diff
@@ -35,11 +38,14 @@ public RemotingSpec(ITestOutputHelper helper) : base(GetConfig(), helper)
             var conf = c2.WithFallback(c1);  //ConfigurationFactory.ParseString(GetOtherRemoteSysConfig());
 
             _remoteSystem = ActorSystem.Create("remote-sys", conf);
+            InitializeLogger(_remoteSystem);
             Deploy(Sys, new Deploy(@"/gonk", new RemoteScope(Addr(_remoteSystem, "tcp"))));
             Deploy(Sys, new Deploy(@"/zagzag", new RemoteScope(Addr(_remoteSystem, "udp"))));
 
             _remote = _remoteSystem.ActorOf(Props.Create<Echo2>(), "echo");
             _here = Sys.ActorSelection("akka.test://remote-sys@localhost:12346/user/echo");
+
+            AtStartup();
```

Comment:
Have to call it here, since we can't cleanly override `base.AtStartup` and have it called via base class constructor - throws a `NullReferenceException` since `_remoteSystem` isn't initialized until concrete constructor finishes being called.